validation-test/SIL/verify_all_overlays.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $x | %FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $x > /dev/null; done
+// RUN: for x in %platform-sdk-overlay-dir/*.swiftmodule; do [[ $(basename "$x") = Swift.swiftmodule ]] && continue; llvm-bcanalyzer $(if [[ -d "$x" ]]; then echo "$x/%target-swiftmodule-name"; else echo "$x"; fi)  | %FileCheck %s; %target-sil-opt -sdk %sdk -enable-sil-verify-all $(if [[ -d "$x" ]]; then echo "$x/%target-swiftmodule-name"; else echo "$x"; fi) > /dev/null; done
```

Comment:
I think that this test would previously work with coreutils from GNUWin32, but the subshell syntax will cause a regression.